src/app/find-help/translation-search/translation-search.component.scss

```diff
@@ -0,0 +1,3 @@
+h1 {
+  text-align: center;
```

Comment:
może by tak wrzucic to do jakies wspolnej klasy dla search, albo text-align-center bootstrapowe?